index.html

```diff
@@ -1,21 +1,139 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="/css/main.css">
+    <title>DESIGN 2: Resume(Responsive grid project)</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+    <div class="container">
+        <header>
+            <div class="header-headline">
+                <h1>
+                    <span> <i class="fa-solid fa-square"></i></span> Maya Nelson
+                </h1>
+                <p>
+                    <span class="forward-slash">&#47 </span>
+                    PROJECT MANAGER
+                </p>
+            </div>
+
+            <div class="nav-button">
+                <button class="nav-toggle" id="navToggle">
+                    <i class='fa-solid fa-bars-staggered'></i>
+                </button>
+            </div>
+            <nav>
+                <ul>
+                    <li <a href=""></a> ABOUT ME </li>
+                    <li>RESUME </li>
+                    <li>PROJECTS</li>
+                    <li>CONTACT </li>
+                </ul>
+            </nav>
+        </header>
+
+        <main>
+            <div class="first-section">
+                <section class="back"> </section>
+                <section class="front">
+                    <div><img src="/designs/maya_nelson.png" alt="maya nleson profile image"></div>
+                    <div>
+                        <h2> Maya
+                            <br>
+                            Nelson
+                        </h2>
+                    </div>
+                    <div class="border">
+                    </div>
+
+                    <div>
+                        <button id="resume-btn">RESUME</button>
```

Comment:
you could also use an `<a>` tag here instead of a button to link to your resume page